app/models/nsm_claim.rb

```diff
@@ -0,0 +1,4 @@
+class NsmClaim < ApplicationRecord
+  has_many :payment_requests, as: :payable
```

Comment:
Are they payable? They're _requests_ to be paid, I think leaving the relationship as-is is fine

Reply:
Also need to add delete/update cascade behavior for the relationships

Reply:
Regarding "payable" it was the most generic naming I could think of for something that needed payments - payment_requestable feels like it doesn't make sense. Also, the alias isn't in there for neatness, I need it to make the payment_request associations polymorphic as it can apply to many different entities (like we have for supportinf documents) 

Invoiceable might be better? but then that might intefere with a future where we link with CCMS and actually send invoices (if we ever have to store those invoices separately to the payment request)

Reply:
That's fair, it's not a huge gripe. Makes enough semantic sense to just be payable